websites/0-9/1anime/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.11",
+	"apiVersion": 1,
+	"author": {
+		"id": "331606403057713154",
+		"name": "faetalize"
+	},
+	"service": "1Anime",
+	"description": {
+		"en": "A presence integrating the famous 1anime streaming website!"
```

Comment:
For description, use the official description from the website

Reply:
Done.